backend/tools/query_slots_with_filter.py

```diff
@@ -0,0 +1,14 @@
+from db.models import Slots
+from sqlalchemy import and_, or_
+
+
+def query_slots_with_filter(date, end_interval, none_or_not):
```

Comment:
def query_slots_with_filter(date, end_interval, none_or_not=None):
and can rename this function: query_slots(....)